controller/controller.go

```diff
@@ -0,0 +1,124 @@
+package controller
+
+import (
+	"compress/gzip"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/brave/go-sync/auth"
+	"github.com/brave/go-sync/command"
+	"github.com/brave/go-sync/datastore"
+	"github.com/brave/go-sync/sync_pb"
+	"github.com/brave/go-sync/timestamp"
+	"github.com/go-chi/chi"
+	"github.com/golang/protobuf/proto"
+	"github.com/rs/zerolog/log"
+)
+
+// SyncRouter add routers for command and auth endpoint requests.
+func SyncRouter(datastore datastore.Datastore) chi.Router {
+	r := chi.NewRouter()
+	r.Post("/command/", Command(datastore))
+	r.Post("/auth", Auth(datastore))
+	r.Get("/timestamp", Timestamp)
+	return r
+}
+
+func sendJSONRsp(body []byte, w http.ResponseWriter) {
+	w.Header().Set("Content-Type", "application/json")
+	_, err := w.Write(body)
+	if err != nil {
+		log.Error().Err(err).Msg("Write HTTP response body failed")
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+}
+
+// Timestamp returns a current timestamp back to sync clients.
+func Timestamp(w http.ResponseWriter, r *http.Request) {
+	body, err := timestamp.GetTimestamp()
```

Comment:
I feel this can be a stateless protocol, timestamp is unused, and can be forged without consequences.

Reply:
Would remove the Timestamp endpoint.